src/components/ActionBar/ActionBar.jsx

```diff
@@ -23,6 +23,9 @@ class ActionsBar extends PureComponent {
 
   static defaultProps = {
     isVisible: false,
+    isMicrophoneEnabled: false,
```

Comment:
this default props are predefined in reducer

Reply:
I think we should separate our components and store. What do you think about it?